pages/blog/authors/kosta.html

```diff
@@ -0,0 +1,9 @@
+---
+group: blog
+layout: author
+title: Konstantinos (Kosta) Servis
+description: All posts written by Konstantinos (Kosta) Servis
+permalink: /blog/authors/kosta/
+author: Kosta
+class: header-white
+---
```

Comment:
Do we need both `kosta.html` and `kostas.html`?

Reply:
No, delete, this is an artifact